pkgs/test/default.nix

```diff
@@ -72,6 +72,12 @@ with pkgs;
   dhall = callPackage ./dhall { };
 
   makeWrapper = callPackage ./make-wrapper {};
```

Comment:
Nit:
```suggestion
  makeWrapper = callPackage ./make-wrapper { };
```

Reply:
I don't know, I prefer `{}`, the file is mostly full of `{}` and I don't think we have an explicit convention for this anywhere :/

Reply:
```
~/Projects/nixpkgs master ⇡
❯ rg -F '{ }' pkgs/top-level/all-packages.nix | wc -l
10181

~/Projects/nixpkgs master ⇡
❯ rg -F '{}' pkgs/top-level/all-packages.nix | wc -l
1101
```

Also this is how it is preferred by both `nixpkgs-fmt` and `nixfmt` formatters.

Reply:
Fine